content/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization.md

```diff
@@ -38,6 +38,12 @@ Before you can add someone as an outside collaborator on a repository, the perso
 If your organization requires two-factor authentication, all outside collaborators must enable two-factor authentication before accepting your invitation to collaborate on a repository. For more information, see "[AUTOTITLE](/organizations/keeping-your-organization-secure/managing-two-factor-authentication-for-your-organization/requiring-two-factor-authentication-in-your-organization)."
 {% endif %}
 
+{% ifversion not ghae %}
+Outside collaborators bypasses SAML SSO requirements. For more information, see "[AUTOTITLE](https://docs.github.com/en/enterprise-cloud@latest/admin/identity-and-access-management/using-saml-for-enterprise-iam/configuring-saml-single-sign-on-for-your-enterprise)"
+
+Outside collaborators cannot be added to Enterprise Managed Users. For more information, see "[AUTOTITLE](content/admin/identity-and-access-management/using-enterprise-managed-users-for-iam/about-enterprise-managed-users.md)."
+{% endif %}
```

Comment:
I think we might want the wording to be more like this? That said, I wonder if we'll be updating this article shortly anyway in light of https://github.com/github/releases/issues/3591. @mattpollard Was that in your plans?

```suggestion
{% ifversion ghec %}
Outside collaborators cannot be added to a repository owned by a {% data variables.enterprise.prodname_emu_org %}. For more information, see "[AUTOTITLE](/admin/identity-and-access-management/using-enterprise-managed-users-for-iam/about-enterprise-managed-users)."
{% endif %}
```

Reply:
The proposed change suggests to the reader that the restriction is with adding them to a repository only, when they cannot be added to an EMU enterprise at all. See update below.

Reply:
No worries @gennaropalma! In that case maybe we can do something like this. We have docs published for the guest collaborators beta now, so I think we can also add a link to that here.

```suggestion
Outside collaborators cannot be added to an {% data variables.enterprise.prodname_emu_enterprise %}. However, you can grant limited access to users outside your enterprise using the guest collaborator role. For more information, see "[AUTOTITLE](/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/roles-in-an-enterprise#guest-collaborators)."
{% endif %}
```